ui/.babelrc

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
why is this deleted? Did we not need it?

Reply:
next.js uses swc by default (instead of babel) - it's faster. Having a .babelrc turns it off